executor/runcexecutor/executor.go

```diff
@@ -140,6 +143,13 @@ func (w *runcExecutor) Exec(ctx context.Context, meta executor.Meta, root cache.
 	return err
 }
 
+func withDefaultSeccompProfile() containerdoci.SpecOpts {
```

Comment:
@crosbymichael like this? or should I add it to the containerd oci package itself :)

Reply:
It should be just:

`spec, cleanup, err := oci.GenerateSpec(ctx, meta, mounts, id, resolvConf, hostsFile, containerdoci.WithUIDGID(uid, gid), seccomp.WithDefaultProfile)`

Reply:
oh there was already a thing.... wow sorry